Classes/System/Solr/Node.php

```diff
@@ -93,7 +93,7 @@ public static function fromArray(array $configuration): Node
      *
      * @param array  $configuration
      * @param string $name
-     * @throws |UnexpectedValueException
+     * @throws |\UnexpectedValueException
```

Comment:
Using use import is better.
Why is pipe here?

Reply:
Looks like an error to me, I couldn't find a reason for it in the history. But you're right, I should have removed the pipe directly, I've done that now.

I've also added an import for `UnexpectedValueException`